releases/sdk5.md

```diff
@@ -47,3 +45,13 @@ No changes are required for apps using the modular framework.
 
 ### SDK 5.0 (August 2, 2018) ###
 This is the first release in the SDK5 series.
+- Runtime: Unknown licenses will be displayed properly.
+- Runtime: Data model now supports `can_print` and `can_export` properties.
+- Runtime: VERSION file is no longer needed.
+- Modular Framework: Videos from Course apps can now be opened from global search.
+- Modular Framework: Copy option is now available for article images.
+- Modular Framework: Minor fixes to existing presets.
+- Renderer: Several improvements for showing articles in small screens.
+
+### SDK 5.1 (September 13, 2018) ###
+This is the last release in the SDK5 series.
```

Comment:
The older 5.x stay at the bottom, with their original comments, you only need to add a new entry with the newer changes for 5.1. Look at SDK-4 notes for examples.